docker.go

```diff
@@ -1590,6 +1592,11 @@ func containerFromDockerResponse(ctx context.Context, response types.Container)
 		return nil, err
 	}
 
+	// the health status of the container, if any
+	if health := container.raw.State.Health; health != nil {
```

Comment:
Is there any case where State can be nil? If so, we can return the container with a default healthStatus 🤔